quantum/keyboard.c

```diff
@@ -95,6 +95,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #ifdef HD44780_ENABLE
 #    include "hd44780.h"
 #endif
+#ifdef WS0010_ENABLE
+#    include "ws0010.h"
+#endif
```

Comment:
```suggestion
```

Reply:
I should remove the ifdefine from keyboard.c totally?

Reply:
It is not being used for anything in this file. Same goes for the HD44780 header; it looks like it was added in the original PR for no real reason: https://github.com/qmk/qmk_firmware/commit/bbea9dadbcc92c4005188860a44c7b9e2479be2e

Reply:
ok, thanks for the explanation